Amplify/Categories/Auth/AuthCategoryBehavior.swift

```diff
@@ -6,9 +6,9 @@
 //
 
 import Foundation
-#if canImport(AuthenticationServices)
 import AuthenticationServices
```

Comment:
Should this import be moved?

Reply:
Yeah, probably a `#if os(iOS) || os(macOS)` would work just fine.